db/migrate/20221102032744_add_users_and_posts_ref_to_comments.rb

```diff
@@ -0,0 +1,6 @@
+class AddUsersAndPostsRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :users, null: false, foreign_key: true
```

Comment:
* As mentioned above 🔼 Kindly ensure your foreign keys are the same as in the ERD diagram currently yours says `users_id` instead of `author_id`